plugins/markdown/komand_markdown/actions/markdown_to_html/action.py

```diff
@@ -13,13 +14,14 @@ def __init__(self):
         )
 
     def run(self, params={}):
-        inbytes = params.get("markdown")
-        instr = params.get("markdown_string")
-        if not (((instr == None) ^ (inbytes == None)) or ((instr == "") ^ (inbytes == ""))):
-            raise Exception(
-                "Only one of Markdown or Markdown String can be defined"
+        inbytes = params.get(Input.MARKDOWN)
+        instr = params.get(Input.MARKDOWN_STRING)
+        if not (((instr is None) ^ (inbytes is None)) or ((instr == "") ^ (inbytes == ""))):
+            raise PluginException(
+                cause="Input error",
+                assistance="Only one of Markdown or Markdown String can be defined"
                 if instr != inbytes
-                else "You must define one of Markdown or Markdown String."
+                else "You must define one of Markdown or Markdown String.",
             )
         if instr:
             html_string = utils.convert(instr, "md", "html")
```

Comment:
Possible to fix end of this run method to be similar to Markdown to HTML action?

Reply:
How do you mean? This is the markdown to html action - do you mean a different one?